### PR TITLE
Fix issue with ShareExtension.js not executing on iOS

### PR DESCRIPTION
**Pull request by jacksonh**
Returning undefined here fixes issues where pages would fail
to scrape if there was no preview image available.

### CODE REVIEW

File: apple/OmnivoreKit/Sources/Views/SyncingIcon.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jackson Harper on 6/5/22.
+//
+
+import Foundation
```
Comment (satindar) [inline apple/OmnivoreKit/Sources/Views/SyncingIcon.swift:8]: code missing or do we need to delete this file?

Comment (jacksonh) [reply apple/OmnivoreKit/Sources/Views/SyncingIcon.swift:8]: @satindar yeah a little more to come...XCode just added it to the commit accidentally.

File: apple/OmnivoreKit/Sources/Views/ShareExtensionView.swift [hunk 2]
```diff
@@ -188,12 +190,8 @@ public struct ShareExtensionChildView: View {
   }
 
   private func localImage(from: URL) -> Image? {
```
Comment (satindar) [inline apple/OmnivoreKit/Sources/Views/ShareExtensionView.swift:192]: from is kinda confusing as a local variable. maybe url instead

`func localImage(from url: URL) -> Image?`

**Review by satindar — APPROVED** [review]

File: apple/OmnivoreKit/Sources/Services/DataService/Mutations/SavePDF.swift [hunk 3]
```diff
@@ -127,7 +126,13 @@ public extension DataService {
     let selection = Selection<MutationResult, Unions.SaveResult> {
       try $0.on(
         saveError: .init { .error(errorCode: (try? $0.errorCodes().first) ?? .unknown) },
-        saveSuccess: .init { .saved(requestId: requestId, url: (try? $0.url()) ?? "") }
+        saveSuccess: .init {
```
Comment (jacksonh) [inline apple/OmnivoreKit/Sources/Services/DataService/Mutations/SavePDF.swift:129]: I am just adding a comment here to test inline comments for something.

**Pull request merged**